components/org.wso2.carbon.identity.api.server.configs/org.wso2.carbon.identity.api.server.configs.v1/src/main/java/org/wso2/carbon/identity/api/server/configs/v1/core/ServerConfigManagementService.java

```diff
@@ -483,7 +483,7 @@ public void resetRemoteServerConfig(String logType) {
         RemoteServerLoggerData remoteServerLoggerData = new RemoteServerLoggerData();
 
         validateLogType(logType);
-        remoteServerLoggerData.setLogType(logType);
+        remoteServerLoggerData.setLogType(resolveLogType(logType));
```

Comment:
You already validate the log type in 485, don't need to have separate method to resolve it.
```suggestion
        remoteServerLoggerData.setLogType(logType.toUpperCase(Locale.ENGLISH));
```

Reply:
Fixed with https://github.com/wso2/identity-api-server/pull/573/commits/9a4d7545dcbc79eca8e3876aee3881ca5876694b